src/test/TxTests.cpp

```diff
@@ -1190,6 +1190,27 @@ revokeSponsorship(AccountID const& accID, SignerKey const& key)
     return op;
 }
 
+Operation
+clawback(AccountID const& from, Asset const& asset, int64_t amount)
```

Comment:
Yeah I can see how the intent isn't as clear as it can be here. I ended up using `AssetCode` here, and adding a `REQUIRE(asset.type() != ASSET_TYPE_NATIVE);` check above in `TestAccount`.

Reply:
What is the motivation for leaving it as `Asset const&` in `TestAccount`?

Reply:
It's simpler to use the existing `Asset` instead of creating an `AssetCode` off of it. If you feel strongly about this I can change the signature and use a `assetToAssetCode` helper method in the tests.

Reply:
It's test so I don't feel particularly strongly either way. I just worry about things that could be confusing, but I don't think this is so bad.